src/runtime/schema/schema.ts

```diff
@@ -121,7 +121,7 @@ export function create(appState: AppState): SchemaInternal {
       checks() {
         NexusSchema.core.assertNoMissingTypes(appState.assembled!.schema, appState.assembled!.missingTypes)
         if (statefulNexusSchema.state.types.length === 0) {
-          log.warn(Layout.schema.emptyExceptionMessage())
+          log.warn(Layout.schemaModules.emptyExceptionMessage())
```

Comment:
OT: Starting to feel off now, like its homeless. A broader linter/doctor/checks system is needed, someday. No central system there. Just our accumulated pokes at the issue. (e.g. <component>.checks() in assembly).

Reply:
Agreed. I kept it that way because I didn't want to prematurely refactor the module. We might have more stuff coming again into it very shortly as well